opentelemetry-quickstart/src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
 quarkus.application.name=myservice
+quarkus.opentelemetry.enabled=true
+quarkus.opentelemetry.tracer.exporter.otlp.endpoint=http://localhost:4317
+
+quarkus.opentelemetry.tracer.exporter.otlp.headers=Authorization=Bearer my_secret
```

Comment:
I'd suggest adding `quarkus.log.console.format` as well, as per https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/opentelemetry.adoc#create-the-configuration